documentation/IDEasy-testing-linux-on-windows.adoc

```diff
@@ -0,0 +1,155 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+= IDEasy Installation on Ubuntu in WSL2 (Windows Subsystem for Linux)
+
+== Preparation
+
+=== Install WSL
+
+WSL for Windows can be installed through the Windows terminal (PowerShell, Git Bash, or CMD) using the following command:
+
+----
+wsl --install
+----
+
+For more details, refer to: https://learn.microsoft.com/en-us/windows/wsl/install[WSL Installation (Microsoft Documentation)]
+
+=== Install Ubuntu from Microsoft Store
+
+Download Ubuntu from the Microsoft Store:
+https://apps.microsoft.com/detail/9PDXGNCFSCZV?hl=en-us&gl=DE&ocid=pdpshare[Microsoft Store]
+
+== IDEasy Setup via Ubuntu Terminal
+
+=== Prerequisite
+
+Ensure Git is installed on your system.
+Check by running:
+
+```
+
+git --version
+
+```
+
+If Git is not installed, use the following command:
+
+```
+
+sudo apt install git
+
+```
+
+=== Download and Setup IDEasy
+
+Refer to the detailed setup guide: link:setup.adoc[Setup Instructions].
+
+To install IDEasy for Linux via the terminal:
```

Comment:
Here you should add a short description how to open ubuntu on Windows.
Ideally add a screenshot of Microsoft Terminal with Ubuntu as single tab and "Welcome to ubuntu message".